primer/src/Primer/App.hs

```diff
@@ -522,7 +540,8 @@ handleEditRequest actions = do
     go :: (Prog, Maybe GVarName) -> ProgAction -> m (Prog, Maybe GVarName)
     go (prog, mdef) a =
       applyProgAction prog mdef a <&> \prog' ->
-        (prog', (.def) <$> progSelection prog')
+        -- TODO this ignores any selection from a type def selection
```

Comment:
Ah, yes. This is because of https://github.com/hackworthltd/primer/pull/964. I'll work out what to do with that PR, probably either merging it first or closing it and cherry-picking here.

Reply:
Rebased on said PR.